app/views/recipes/show.html.erb

```diff
@@ -1,9 +1,5 @@
 <h1><%= @recipe.title %></h1>
   <div class="container">
-    <%# if @recipe.photo.present? %>
```

Comment:
why don't you just uncomment that instead of deleteing it? @akramerin

Reply:
because right now it is not working (there is not photo part of the recipe schema, so it results in an error) and i can not check IF it is working. i thought, rather than keeping a wrong line in there, delete it.